src/main/java/graphql/normalized/ExecutableNormalizedOperationFactory.java

```diff
@@ -112,7 +151,8 @@ private ExecutableNormalizedOperation createNormalizedQueryImpl(GraphQLSchema gr
                                                                     OperationDefinition operationDefinition,
                                                                     Map<String, FragmentDefinition> fragments,
                                                                     CoercedVariables coercedVariableValues,
-                                                                    @Nullable Map<String, NormalizedInputValue> normalizedVariableValues) {
+                                                                    @Nullable Map<String, NormalizedInputValue> normalizedVariableValues,
+                                                                    GraphQLContext graphQLContext, Locale locale) {
```

Comment:
Nit: can you push `Locale locale` to the next line?
```suggestion
                                                                    GraphQLContext graphQLContext,
                                                                    Locale locale) {
```